packages/openapi-to-graphql/src/schema_builder.ts

```diff
@@ -464,8 +464,9 @@ function createFields({
     // Determine if this property is required in mutations
     const reqMutationProp =
       isMutation &&
-      'required' in schema &&
```

Comment:
I think this is a typo. This is just wrong. What you added...
```
'required' in def.schema && def.schema.required.includes(fieldTypeKey)
```
...is the only thing that should be there. I am currently making the necessary changes.